brevent/src/main/res/values-pl-v26/strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="brevent_about_developer_adb">Nie wyłączaj \"Debugowania USB\" oraz \"Trybu USB\"</string>
```

Comment:
`Trybu USB` -> `konfigurację USB`, as in Development Options.

Reply:
Corrected and checked, thank you.